### ISSUE TITLE
A11y_Azure Data Studio_Azure tree item_Screenreader: Screenreader announce unnecessary content when keyboard focus goes to selected Azure account Tree Item after pressing tab or UP/down arrow key. 

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/azuredatastudio#14172]
“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#Screenreader;#JAWS;#NVDA;#STP;#AINotInScope;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.26.0-insider (user setup)
Windows Version: Windows10
Screenreader: JAWS, NVDA

**Repro Steps:**
1.Launch Azure Data Studio insiders application.
2.Tab to Connections from left menu and in servers button and hit enter on "New Connection" button.
3.Connections section will get open.
4.Tab to "Browse Preview" Tab item and hit enter.
5.Navigate to "Azure account" present in "Azure" tree item with the help of Tab or up/down arrow key.
6.Start JAWS and verify issue.

**Actual:**
Screenreader announce unnecessary content when keyboard focus goes to selected Azure account Tree Item after pressing tab or UP/down arrow key. 

**Same issue is also observed with all the tree items present in this section.**

**Expected:**
After pressing Tab or Up/Down arrow key and move to selected Azure Account tree item , Screenreader should announce account detail only instead of unnecessary detail.

**User Impact:**
Screen reader user will not  get correct information and not ne able to understand which one tree item is selected if screenreader does not announce detail correctly.

**Comment by prabhudevu** [comment]
![14172_Fixed](https://user-images.githubusercontent.com/98821792/216604121-0237dcbd-2e38-42c3-9121-32fac4bfbf2c.png)

I have verified the issue in the version 1.41.1
Issue is not reproducible.

**GithubTags:**#Closed;#Regressed:03-02-23;
